src/child/containers/sidebars/Sidebar.js

```diff
@@ -130,13 +132,18 @@ class Sidebar extends Component {
         }
     }
 
+    openWindow(windowName) {
+        this.props.dispatch(reopenWindow(windowName));
+    }
+
     render() {
-        const { sidebar } = this.props;
+        const { sidebar, unopenedWindows } = this.props;
```

Comment:
`unopenedWindows` seems a little odd to me, would `closedWindows` be a better name?
Unless there's a reason otherwise?